pkg/operator/ceph/csi/spec.go

```diff
@@ -315,6 +318,17 @@ func startDrivers(clientset kubernetes.Interface, rookclientset rookclient.Inter
 		}
 	}
 
+	tp.ProvisionerReplicas = defaultProvisionerReplicas
+	nodes, err := clientset.CoreV1().Nodes().List(metav1.ListOptions{})
+	if err != nil {
+		logger.Errorf("failed to get nodes. Defaulting the number of replicas of provisioner pods to %d. %v",
+			defaultProvisionerReplicas, err)
+	}
```

Comment:
If there was an error, `nodes` would be nil, so we need to put what's next in an `else` block, right?